.gitignore

```diff
@@ -0,0 +1,5 @@
+build/
```

Comment:
@kotylamichal do we need this?

Reply:
It is useful, this makes `git status` output more clear. This will also help us to avoid situations when someone unintentionally commits changes in cloned repo (like `meta-trenchboot`) or build files

Reply:
@kotylamichal shouldn't these dirs be outside of the meta-fobnail layer?